tests/unit/test_local_session.py

```diff
@@ -172,6 +172,33 @@ def test_describe_model(LocalSession):
     assert response['PrimaryContainer']['ModelDataUrl'] == '/some/model/path'
 
 
+@patch('sagemaker.local.local_session._LocalTransformJob')
+@patch('sagemaker.local.local_session.LocalSession')
+def test_create_transform_job(LocalSession, _LocalTransformJob):
+    local_sagemaker_client = sagemaker.local.local_session.LocalSagemakerClient()
+
+    local_sagemaker_client.create_transform_job('transform-job', 'some-model', None, None, None)
+    _LocalTransformJob().start.assert_called()
+
+    local_sagemaker_client.describe_transform_job('transform-job')
+    _LocalTransformJob().describe.assert_called()
+
+
+@patch('sagemaker.local.local_session._LocalTransformJob')
+@patch('sagemaker.local.local_session.LocalSession')
+def test_describe_transform_job(LocalSession, _LocalTransformJob):
+    local_sagemaker_client = sagemaker.local.local_session.LocalSagemakerClient()
+
+    local_sagemaker_client.create_transform_job('my-transform-job', 'some-model', None, None, None)
+    _LocalTransformJob().start.assert_called()
+
+    local_sagemaker_client.describe_transform_job('my-transform-job')
+    _LocalTransformJob().describe.assert_called()
```

Comment:
is the only difference between this test and the one above the added failure case?

Reply:
yeah, got rid of the repeated code here as it makes no sense.